package.json

```diff
@@ -15,7 +15,7 @@
 	},
 	"devDependencies": {
 		"chrome-webstore-upload-cli": "^1.2.0",
-		"copy-webpack-plugin": "^5.0.3",
+		"copy-webpack-plugin": "5.0.3",
```

Comment:
Let's fix this in this PR as per https://github.com/notlmn/browser-extension-template/issues/22#issuecomment-570781123, or will have to insert package-lock.json again? WDYT?

Reply:
I don't know how much more work is needed here, so it's best to fix that separately.